packages/useTrie/src/index.ts

```diff
@@ -55,21 +57,22 @@ class Trie implements ITrie {
     words.forEach(word => this.add(word, this.getText));
   }
 
-  private normalizeWord = (word: Word) =>
```

Comment:
I've been too used to arrow methods thanks to Class Components.
This change makes the code look readable with less symbols (less cognitive overload)